docs/user/resources/04-tracepipeline.md

```diff
@@ -136,13 +136,13 @@ The status of the TracePipeline is determined by the condition types `GatewayHea
 | GatewayHealthy         | True             | GatewayReady                | Trace gateway Deployment is ready                                                                                                                                                                                       |
-| GatewayHealthy         | True             | GatewayReady                | Pods are being started/updated                                                                                                                                                                                          |
+| GatewayHealthy         | True             | RolloutInProgress           | Pods are being started/updated                                                                                                                                                                                          |
 | GatewayHealthy         | False            | GatewayNotReady             | No Pods deployed                                                                                                                                                                                                        |
 | GatewayHealthy         | False            | GatewayNotReady             | Failed to list ReplicaSets: `reason`                                                                                                                                                                                    |
 | GatewayHealthy         | False            | GatewayNotReady             | Failed to fetch ReplicaSets: `reason`                                                                                                                                                                                   |
 | GatewayHealthy         | False            | GatewayNotReady             | Pod is not scheduled: `reason`                                                                                                                                                                                          |
-| GatewayHealthy         | False            | GatewayNotReady             | Pod is in the pending state as container: `container name` is not running due to: `reason`                                                                                                                                  |
-| GatewayHealthy         | False            | GatewayNotReady             | Pod is in the failed state due to: `reason`                                                                                                                                                                                 |
+| GatewayHealthy         | False            | GatewayNotReady             | Pod is in the pending state as container: `container name` is not running due to: `reason`                                                                                                                              |
```

Comment:
```suggestion
| GatewayHealthy         | False            | GatewayNotReady             | Pod is in the pending state because container: `container name` is not running due to: `reason`                                                                                                                              |
```